tests/ci/cdk/util/iam_policies.py

```diff
@@ -202,3 +202,32 @@ def ecr_power_user_policy_in_json(ecr_repo_names):
             }
         ]
     }
+
+def device_farm_access_policy_in_json():
+    """
+    Define an IAM policy statement for Device Farm operations.
+    :return: an IAM policy statement in json.
+    """
+    resources = []
+    resources.append("arn:aws:devicefarm:{}:{}:*:*".format(AWS_REGION, AWS_ACCOUNT))
```

Comment:
Nit: Do we have more constrained resource naming instead of `*:*` to minimize the permission? e.g. `aws-lc-ci:*`.

https://docs.aws.amazon.com/devicefarm/latest/developerguide/artifacts.html
```
"arn": "arn:aws:devicefarm:us-west-2:123456789101:test:5e01a8c7-c861-4c0a-b1d5-12345EXAMPLE",
```

Reply:
The first `*` is for the different types of device farm resources (jobs, projects, device pools, runs, etc), which we will all need.  Unfortunately, the second `*` is not a simple project name that we can assign ourselves like `aws-lc-android-ci`, it points to a hash representing the project. The device farm project hash id which would have to be manually changed each time we update to a new device farm project.